Source/WebCore/rendering/RenderTableSection.cpp

```diff
@@ -569,9 +570,9 @@ void RenderTableSection::layoutRows()
         if (RenderTableRow* rowRenderer = m_grid[r].rowRenderer) {
             // FIXME: the x() position of the row should be table()->hBorderSpacing() so that it can 
             // report the correct offsetLeft. However, that will require a lot of rebaselining of test results.
-            rowRenderer->setLogicalLeft(0_lu);
+            rowRenderer->setLogicalLeft(0_lu + hspacing);
```

Comment:
We can merge `left` and `top` with and it will account for horizontal writing mode as well:

`rowRenderer->setLogicalLocation({ 0, m_rowPos[r]});`